mesonbuild/compilers/mixins/clang.py

```diff
@@ -109,12 +109,6 @@ def openmp_flags(self) -> T.List[str]:
             # Shouldn't work, but it'll be checked explicitly in the OpenMP dependency.
             return []
     
-    def get_win_subsystem_args(self, value: str) -> T.List[str]:
-        if self.info.is_windows() and not self.info.is_cygwin() and isinstance(self.linker, ClangClDynamicLinker):
-            return [f'-Wl,/subsystem:{value}']
```

Comment:
I see that the code that ended up breaking lld for mingw targets was moved from here. The bit that was lost in the move was the `isinstance(self.linker, ClangClDynamicLinker)` bit, which is why it worked before.